ci-operator/step-registry/assisted/vsphere/setup/assisted-vsphere-setup-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: assisted-vsphere-setup
+  steps:
+    - ref: openshift-cluster-bot-rbac
```

Comment:
cluster-bot? what does it have to do with this workflow?

Reply:
It used for Openshift for testing but can be eliminated